website/server/controllers/api-v3/groups.js

```diff
@@ -718,4 +715,19 @@ api.inviteToGroup = {
   },
 };
 
+// Transitioning invitation storage to an array of objects (see issue #7792 for reference)
+function clearPartyInvitation(user, groupID) {
+  user.invitations.parties.forEach(function(party, index, partiesArr) {
```

Comment:
It doesn't need markModified because it uses array methods that allow mongoose to keep track of what needs to be updated.